Pearson-Advance/openedx-lti-tool-plugin
# PADV-592: Add LtiGradedResource model

## Ticket

https://agile-jira.pearson.com/browse/PADV-592

## Description

This PR adds a model to store information about graded resources for the LTI AGS service. This also includes a custom model manager with method to retrieve graded resources for a specific user_id, this will be used on future PRs to retrieve graded resources from edx-platform grading signals.

## Type of Change

- [x] Add LtiGradedResourceManager model manager class.
- [x] Add LtiGradedResource model.
- [x] Include tests for all added code.

## Testing:

- Run the LMS: `make dev.up.lms`.
- Install `openedx_lti_tool_plugin` on the LMS.
- Run migrations: `./manage.py lms migrate`.
- Run LMS shell: `./manage.py lms shell`.
- The LtiGradedResource model should be available to import from the openedx_lti_tool_plugin app.

## Reviewers
